server/src/main/java/com/adobe/testing/s3mock/FileStoreController.java

```diff
@@ -216,9 +217,17 @@ public ListAllMyBucketsResult listBuckets() {
     if (s3Object != null) {
       final HttpHeaders responseHeaders = new HttpHeaders();
       responseHeaders.setContentLength(Long.valueOf(s3Object.getSize()));
-      if (!"".equals(s3Object.getContentType())) {
-        responseHeaders.setContentType(MediaType.parseMediaType(s3Object.getContentType()));
+
+      MediaType fileMediaType;
+
+      try {
+        fileMediaType = MediaType.parseMediaType(s3Object.getContentType());
+      } catch (InvalidMediaTypeException e) {
+        fileMediaType = new MediaType("binary","octet-stream");
```

Comment:
missing whitespace after `,`:
```suggestion
        fileMediaType = new MediaType("binary", "octet-stream");
```